.github/workflows/phpunit.yml

```diff
@@ -20,6 +20,10 @@ jobs:
       GCNOTIFY_TEMPLATE_NEW_JOB_POSTED_EN: ${{ vars.GCNOTIFY_TEMPLATE_NEW_JOB_POSTED_EN }}
       GCNOTIFY_TEMPLATE_NEW_JOB_POSTED_FR: ${{ vars.GCNOTIFY_TEMPLATE_NEW_JOB_POSTED_FR }}
       GCNOTIFY_TEMPLATE_TALENT_REQUEST_SUBMISSION_CONFIRMATION_ENFR: ${{ vars.GCNOTIFY_TEMPLATE_TALENT_REQUEST_SUBMISSION_CONFIRMATION_ENFR }}
+      GCNOTIFY_TEMPLATE_NOMINATION_RECEIVED_SUBMITTER_EN: ${{ vars.GCNOTIFY_TEMPLATE_NOMINATION_RECEIVED_SUBMITTER_EN }}
+      GCNOTIFY_TEMPLATE_NOMINATION_RECEIVED_SUBMITTER_FR: ${{ vars.GCNOTIFY_TEMPLATE_NOMINATION_RECEIVED_SUBMITTER_FR }}
```

Comment:
we need to think about some alternative for this .. lol! It keeps accumulating s lot.